src/common/hashes/index.ts

```diff
@@ -84,36 +84,36 @@ export const computeBinaryTransactionSigningHash = (
 }
 
 /**
- * Compute Account Ledger Object ID
+ * Compute Account Ledger Object index
  *
- * All objects in a ledger's state tree have a unique ID.
- * The Account Ledger Object ID is derived by hashing the
+ * All objects in a ledger's state tree have a unique index.
+ * The Account Ledger Object index is derived by hashing the
  * address with a namespace identifier. This ensures every
- * ID is unique.
+ * index is unique.
  *
  * See [Ledger Object IDs](https://xrpl.org/ledger-object-ids.html)
  *
  * @param address The classic account address
- * @returns {string} The Ledger Object ID for the account
+ * @returns {string} The Ledger Object Index for the account
  */
-export const computeAccountLedgerObjectID = (address: string): string => {
+export const computeAccountLedgerObjectIndex = (address: string): string => {
```

Comment:
```suggestion
export const computeAccountRootIndex = (address: string): string => {
```
The ledger object part is implied, and just makes the function name super long/annoying to type